src/components/hmi_message_handler/src/mqueue_adapter.cc

```diff
@@ -70,8 +72,13 @@ class ReceiverThreadDelegate : public threads::ThreadDelegate {
     }
   }
 
+  void exitThreadMain() OVERRIDE {
+    shutdown_requested_ = true;
```

Comment:
@pestOO Fixed, see 9b1e851350176ae8bf460d6467fab2f40cf8804b Please note that `MqueueAdapter` is not used anywhere and related Unit tests are disabled with comment to be fixed in a separate task from @Kozoriz